openedx/features/philu_utils/backend_storage.py

```diff
@@ -37,7 +39,7 @@ def _s3_client(self):
             aws_secret_access_key=self.aws_secret_access_key,
         )
 
-    def _get_available_name(self, name, max_length=None):
+    def _get_available_name(self, name, max_length=None):  # pylint: disable=unused-argument
```

Comment:
I updated the docstring.

I suppress this warning because i thought that it is overridden method, But I'm not sure. 
@zahid-arbisoft can better tell that removing unused `max_length` parameter can cause any side effect or not.

Reply:
I and good either way; though this is not an overridden method. Methods starting with an underscore are not overridden.